server/node-server.js

```diff
@@ -51,8 +45,10 @@ app.get('/auth/facebook/callback',
     session: false,
   }),
   (request, response) => {
-    response.redirect('/auth/facebook?access_token=' + request.user.token
-     + '&display_name=' + request.user.displayName);
+    const { token, displayName } = request.user;
+    response.redirect(
+      `/login/facebook?access_token=${token}&display_name=${displayName}`
```

Comment:
I was getting an error because I was using a `localhost:3000` in my callback url, and it was redirecting to the server's `/auth/facebook` route, instead of the react `/auth/facebook` route. Changing the address to `localhost:8080` fixed the problem, but I decided to change the route because in production, we would have had the front-end and the backend running off the same server + port, which would have led to undefined behavior when we hit `/auth/facebook`.